modules/Hoot/ui/modals/grailDatasetPicker.js

```diff
@@ -27,7 +27,6 @@ export default class GrailDatasetPicker {
                     readonly: 'readonly',
                     sort: false,
                     itemKey: 'name',
-                    _value: this.layer.id,
```

Comment:
How come you removed this?  `_value` is the hack to provide the combobox with a value that differs from the option label.

Reply:
it was giving it a default value which would be the layer itself so when you open the combobox and press enter, even though you didnt select an option it would go to the submit function call, retrieve the _value, and get that default.

The combobox still looks the same as far as I can see